### DISCUSSION TITLE
Peeko

### DISCUSSION THREAD

**Discussion by KittyMasta** [CRobbins0867/ProdigyHacking--MobileHacking#38] [General]
A long time ago i played prodigy and i had a Peeko but then i forgot my account the update then came and peeko was gone... for ever.. ): so guys i wondered if you could help by either making a hacked account for peeko or giving me a account that has peeko or telling me how to hack to get peeko!

**Comment by SakuraKitsunee** [comment]
@KittyMasta sorry for the delay, is it okay if it's level one, because you can change the pet level on the mod menu right now

**Comment by KittyMasta** [comment]
Oh well i was wondering when you will respond to making me my microsoft i really want one

**Comment by ShawnC32** [comment]
Peeko is possible to gain when you click on get all pets
